packages/adyen-integration/jest.config.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
     displayName: 'adyen-integration',
     preset: '../../jest.preset.js',
-    globals: {
-        'ts-jest': {
+    transform: {
+        '^.+\\.[tj]sx?$': ['ts-jest', {
```

Comment:
Need to move ts-jest config to transform as that configuration in globals is deprecated. 
https://stackoverflow.com/questions/75955311/warn-define-ts-jest-config-under-globals-is-deprecated